dependencies/src/test/resources/exclusion-sample-rules/target-one-match.xml

```diff
@@ -0,0 +1,12 @@
+<LinkageCheckerFilter>
+  <LinkageError>
+    <Target>
+      <Method className="com.google.Foo" name="methodA" />
```

Comment:
There's a combinatorial explosion here: every source class against every target method/field. Perhaps each target should have at most one source and at most one target and each target can have exactly one child.

Reply:
Created a followup on the restriction to avoid combinational explosion.

https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1278

Reply:
This is not really followup eligible. We need to get this right in the design now, not fix it later.